inference-engine/tests/functional/inference_engine/variable_state.cpp

```diff
@@ -31,3 +33,25 @@ TEST(VariableStateCPPTests, throwsOnUninitializedSetState) {
     Blob::Ptr blob;
     ASSERT_THROW(req.SetState(blob), InferenceEngine::NotAllocated);
 }
+
+
+TEST(VariableStateOVTests, throwsOnUninitializedReset) {
```

Comment:
Can we create separate files for new tests?

Reply:
removed